engine/api/src/main/java/org/apache/cloudstack/engine/subsystem/api/storage/DataStoreManager.java

```diff
@@ -33,7 +33,11 @@
 
     List<DataStore> getImageStoresByScope(ZoneScope scope);
 
-    DataStore getImageStore(long zoneId);
+    DataStore getImageStoreForRead(long zoneId);
```

Comment:
`getRandomImageStore` or something that conveys meaning? You may also put javadocs on them to convey which method does what?